README.md

```diff
@@ -82,11 +82,10 @@ dsv policy create \
   --actions 'read' \
   --effect 'allow' \
   --subjects "roles:$rolename" \
```

Comment:
```suggestion
  --subjects "roles:$rolename" \
  --desc "${desc}" \
```

I want to leave description in for creation of secret, even if it was put in a codeblock above as it was intended to be done all in one session. cheers!